package.tmpl

```diff
@@ -110,10 +108,13 @@ func {{.GoName}}({{template "paramsGoDecl" .Parameters}}){{if not .Return.IsVoid
 {{end}}
 
 func Init() error {
-  return InitWithProcAddrFunc(auto.GetProcAddress)
+  return InitWithProcAddrFunc(getProcAddress)
 }
 
-func InitWithProcAddrFunc(getProcAddr procaddr.GetProcAddressFunc) error {
+// InitWithProcAddrFunc intializes the package with the given getProcAddr
```

Comment:
consider a slight doc simplification: "InitWithProcAddrFunc initializes the package using the specified OpenGL function pointer loading function. For most cases Init should be used instead."